.gitignore

```diff
@@ -17,7 +17,6 @@ inst/
 .vscode/*
 !.vscode/settings.json
 !.vscode/tasks.json
-!.vscode/launch.json
```

Comment:
Just curious - why remove this line?

Reply:
@HansKallekleiv  As I understand .gitignore, all matching the patterns in there is not committed. However, using ! as a negator, those lines are exceptions for the afore written patterns. This means that we would not commit anything inside .vscode/ except for settings.json, tasks.json and launch.json. Do we want to commit userspecific VS Code settings? I was assuming not, but in that case we should also remove the other exceptions. What do you think, @asnyv?

Reply:
Seems like it is quite a common thing (and not something we have added manually): https://stackoverflow.com/a/32979933
Mainly a question of preferences I guess: should user settings, debug setup and etc be shared among us. 

I am happy with removing all of them, but also ok with keeping them as is.

Reply:
I put it back in again since we also keep the other ones. We can decide another time and remove it in another PR if we conclude that we don't need it.